deploy/python/linux/roles/instrument/tasks/main.yml

```diff
@@ -26,6 +26,22 @@
     replace: '{{ service_display_name }}'
     backup: yes
 
+- name: Format tags into a list

```

Comment:
@Julien4218 I saw that you put a comment on the java PR for the tags...

`tags: {"dxOwningTeam":"DemoX","dxEnvironment":"development",...} JSON list of tags`

Should we add the same comment here?

Reply:
Sure, I just added it.